### ISSUE TITLE
--no-delete-output-path got removed?

### ISSUE THREAD

**Issue by luksireiku** [angular/angular-cli#6459]
### Bug Report or Feature Request (mark with an `x`)
```
- [x] bug report -> please search issues before submitting
- [ ] feature request
```

### Versions.

Your global Angular CLI version (1.0.3) is greater than your local
version (1.0.0). The local Angular CLI version is used.

### Repro steps.

`ng build --no-delete-output-path`

### The log given by the failure.

```
The option '--no-delete-output-path' is not registered with the build command. Run `ng build --help` for a list of supported options.
```

### Desired functionality.

To preserve the /dist folder but overwrite it's contents

**Comment by clydin** [comment]
The option is a new feature that will be available in version 1.1.  You can try it out now via the latest release candidate (1.1.0-rc.1).

**Comment by sumitarora** [comment]
Closing as above.
